### PR TITLE
reconcile title and id in PerPassageEmbeddings

### PR DESCRIPTION
**Pull request by zzzming**
Replace the "title" field in the PerPassageEmbeddings and use "id" stead from the last code review.

### CODE REVIEW

File: ragstack/colbert/token_embedding.py [hunk 1]
```diff
@@ -12,13 +12,11 @@ class PerTokenEmbeddings:
 
     def __init__(
         self,
-        id: int,
-        part: int,
-        parent_id: uuid.UUID = None,
-        title: str = "",
+        id: int,  # id is the embedding id
+        part: int,  # part is the part id of the passage
+        title: str = "",  # title is the title of the passage
```
Comment (epinzur) [inline ragstack/colbert/token_embedding.py:17]: i think we should drop `title` completely from this code. If someone wants to store the name of the file or a document `title`, they will do it in the metadata.

File: ragstack/colbert/token_embedding.py [hunk 2]
```diff
@@ -43,26 +41,32 @@ class PassageEmbeddings:
...
+    __model: str
+    __dim: int
 
     def __init__(
         self,
         text: str,
-        title: str = "",
-        part: int = 0,
-        id: uuid.UUID = None,
+        title: str = "",  # keep this as backward compatibility, use id instead
```
Comment (epinzur) [inline ragstack/colbert/token_embedding.py:51]: do we need to worry about backward compatibility? this hasn't been released yet, correct?

**Review by epinzur — CHANGES_REQUESTED** [review]

File: ragstack/colbert/token_embedding.py (hunk 2 above)
Comment (nicoloboschi) [reply ragstack/colbert/token_embedding.py:51]: nope

**Comment by epinzur** [comment]
merged https://github.com/datastax/ragstack-ai/pull/347 instead of this. closing the PR.

**Pull request closed without merging**